### PR TITLE
More accessibility options and pref menu

### PR DESCRIPTION
**Pull request by grungussuss**
## About The Pull Request
- added accessibility tab in prefs menu
- put darkened flashes option into this menu
- added `darken screen shake` pref, for motion sick people, darkens your screen when experiencing screen shake.

<details>
OUTDATED 

image_2024-10-29_023038297

</details>

https://github.com/user-attachments/assets/e3fd43ad-a65c-47dc-add2-6c93048e8b61

## Why It's Good For The Game

makes it easier for people with disabilities to play the game.
## Changelog
:cl: grungussuss
qol: you can now mitigate motion sickness from screen shake by enabling "darken screen shake" in preferences
/:cl:

### CODE REVIEW

File: code/game/sound.dm [hunk 1]
```diff
@@ -82,6 +83,8 @@
 			listeners += get_hearers_in_view(maxdistance, below_turf)
 
 	for(var/mob/listening_mob in listeners | SSmobs.dead_players_by_zlevel[source_z])//observers always hear through walls
+		if(listening_mob.client?.prefs.read_preference(preference))
```
Comment (MrMelbert) [inline code/game/sound.dm:86]:
```suggestion
		if(preference && listening_mob.client?.prefs.read_preference(preference))
```

**Review by MrMelbert — COMMENTED** [review]
Someone has historically been against putting surgery sounds behind prefs, not sure who, maybe lemon or oranges?

**Comment by OrionTheFox** [comment]
Don't we also have settings for breathing/heartbeat sounds? Might fit to move those here too.

**Comment by MoonDeer728** [comment]
> Don't we also have settings for breathing/heartbeat sounds? Might fit to move those here too.

breathing switch is in prefs

**Comment by Sadboysuss** [comment]
> Someone has historically been against putting surgery sounds behind prefs, not sure who, maybe lemon or oranges?

@LemonInTheDark @optimumtact 

are you against this

**Comment by LemonInTheDark** [comment]
I think it's kinda silly, surgery sounds are meant to be a bit uncomfortable and I don't like the idea that the game can/should be truly comfortable. 
Would need better justification then none but I am by default against it.

**Comment by Sadboysuss** [comment]
After talking with lemon, we have decuded to remove the surgery sounds pref, I'll make the change later

**Comment by optimumtact** [comment]
It was me who was against it, because it encourages the wrong kind of thinking about the game where you will deal with quite adult themes

**Pull request merged**